.github/workflows/verify.yml

```diff
@@ -0,0 +1,65 @@
+name: Certora
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - certora/erc20
+      - certora/erc1155ext
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install python
+        uses: actions/setup-python@v2
+        with: { python-version: 3.6, cache: 'pip' }
+
+      - name: Install java
+        uses: actions/setup-java@v1
+        with: { java-version: "11", java-package: jre }
+
+      - name: Install certora
+        run: pip install certora-cli
+
+      - name: Install solc
+        run: |
+          wget https://github.com/ethereum/solidity/releases/download/v0.8.4/solc-static-linux
```

Comment:
solidity version ?

Reply:
version 0.8.4 is specified in the github link

Reply:
Sorry my message was not clear (it was mostly an internal post-it of what we have to discuss). The real question in the line of:
- do we want to have a solc version hardcoded ? 
- was processes do we put in place to maintain that ?
- does this version have to match the scripts ?